tree/30_generic_methods/service_check_running_src.cf

```diff
@@ -0,0 +1,56 @@
+#####################################################################################
+# Copyright 2015 Normation SAS
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# @name Service check running with SRC
+# @description Check if a service is running using SRC (AIX System Resource Controller)
+#
+# @parameter service_name Subsystem name in SRC
+#
+# @class_prefix service_check_running
+# @class_parameter service_name
+# This bundle will define a class service_check_running_${service_name}_{kept,not_ok}
+
+bundle agent service_check_running_src(service_name)
```

Comment:
They should most definitely not go into 20_, because that is for body libraries.

The use of "_" would be ideal here, like _service_check_running_src. This hides methods from users.

However, we already a couple like this in ncf today:

* service_check_running_ps.cf
* service_ensure_running_path.cf